src/Console/Formatter/OutputFormatter.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Silverorange\PackageRelease\Console\Formatter;
+
+use Symfony\Component\Console\Formatter\OutputFormatter as SymfonyOutputFormatter;
+
+/**
+ * Custom output formatter that works with mbstring function overloading
+ * enabled
+ *
+ * See https://github.com/symfony/symfony/pull/26101/
+ *
+ * @package   PackageRelease
+ * @author    Michael Gauthier <mike@silverorange.com>
+ * @copyright 2018 silverorange
+ * @license   http://www.opensource.org/licenses/mit-license.html MIT License
+ */
+class OutputFormatter extends SymfonyOutputFormatter
+{
+    public function format($message)
+    {
+        $message = (string) $message;
+        $offset = 0;
+        $output = '';
+        $tagRegex = '[a-z][a-z0-9,_=;-]*+';
+        // Note: Offset capture is in bytes. For compatibility, explicitly use
+        // mbstring with 8bit encoding for string manipulations.
+        preg_match_all("#<(($tagRegex) | /($tagRegex)?)>#ix", $message, $matches, PREG_OFFSET_CAPTURE);
+        foreach ($matches[0] as $i => $match) {
+            $pos = $match[1];
+            $text = $match[0];
+
+            if (0 !== $pos && '\\' == mb_substr($message, $pos - 1, 1, '8bit')) {
+                continue;
+            }
+
+            // add the text up to the next tag
+            $output .= $this->applyCurrentStyle(mb_substr($message, $offset, $pos - $offset, '8bit'));
+            $offset = $pos + mb_strlen($text, '8bit');
+
+            // opening tag?
+            if ($open = '/' != mb_substr($text, 1, 1, '8bit')) {
+                $tag = $matches[1][$i][0];
+            } else {
+                $tag = isset($matches[3][$i][0]) ? $matches[3][$i][0] : '';
+            }
+
+            if (!$open && !$tag) {
```

Comment:
I just want to check that I'm understanding this right: `<output>bla</output>` is the same as `<output>bla</>` and it's valid syntax to go like `<output>bla<header> bla bla</output></header>`?

Reply:
OK I read the PR in the header, this is the standard `SymfonyOutputFormatter` updated to use mbstring. 👍